src/ubuntu/18.04/mono/amd64/Dockerfile

```diff
@@ -15,6 +15,7 @@ RUN apt-get update && \
         gdb \
         gettext \
         git \
+        jq \
```

Comment:
Slightly out of order. Needs to be moved to be under `iputils-ping`. Also applies to the other Ubuntu 18.04 Dockerfiles.

Reply:
Thanks, will update once https://github.com/dotnet/arcade/pull/10175.goes in. :)

Reply:
Fixed.